packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -19,15 +23,20 @@ export function GraphQLUniversalProvider({children, createClient}: Props) {
       ReturnType<typeof createSsrExtractableLink>
     ]
   >(() => {
+    const clientOptions = createClientOptions();
     const link = createSsrExtractableLink();
-    const client = createClient();
-    client.link = link.concat(client.link);
+
+    if (clientOptions.link) {
+      clientOptions.link = link.concat(clientOptions.link);
```

Comment:
You shouldn't mutate their original object, and shouldn't you still assign `clientOptions.link` to `link` if they didn't have an explicit one?

Reply:
Don't we want to provide the `createSsrExtractableLink` link as we were doing before? 

> shouldn't you still assign clientOptions.link to link if they didn't have an explicit one?

Will fix

Reply:
This should be `if (!clientOptions.link)` then, right?

Reply:
no, like, if they have no link, add one. If they have one, augment it.